docker/package/tezos_setup_wizard.py

```diff
@@ -214,9 +214,14 @@ def yes_or_no(prompt, default=None):
             print(color("Please provide a 'yes' or 'no' answer.", "\x1b[1;31m"))
 
 
-def list_connected_ledgers():
-    output = get_proc_output("sudo -u tezos -E tezos-client list connected ledgers")
-    return [name.decode() for name in re.findall(ledger_regex, output.stdout)]
+def wait_for_ledger_baking_app():
+    output = b""
+    while re.search(b"Found a Tezos Baking", output) is None:
+        output = get_proc_output(
+            "sudo -u tezos -E tezos-client list connected ledgers"
+        ).stdout
+        proc_call("sleep 1")
+    return [name.decode() for name in re.findall(ledger_regex, output)]
```

Comment:
Is there a reason why you opted for the `while` loop and not just moved the `input` prompt? Do you think this way will be more resilient? I'm asking because this calls external processes (`tezos-client` and `sleep 1`) potentially a lot and potentially unnecessary, and that feels a bit iffy.

But if you think this is better, then otherwise looks good.

Reply:
The main reason was to force the user to open the Baking app before proceeding the setup just to be sure that we'll be able to fetch ledger-stored keys

Reply:
I was following the Kiln behaviour, which tells to open the Baking app on the ledger and then waits till its open